Sources/NIOCore/NIOLoopBound.swift

```diff
@@ -142,3 +142,26 @@ public final class NIOLoopBoundBox<Value>: @unchecked Sendable {
         }
     }
 }
+
+#if compiler(>=6.0)  // `sending` is >= 6.0
+extension NIOLoopBoundBox {
+    /// Initialise a ``NIOLoopBoundBox`` by `sending` (i.e. transferring) a value, validly callable off `eventLoop`.
+    ///
+    /// Contrary to ``init(_:eventLoop:)``, this method can be called off `eventLoop` because we are `sending` the value across the isolation domain.
+    /// Because we're `sending` `value`, we just need to protect it against mutations (because nothing else can have access to it anymore).
+    public static func makeBoxSendingValue(
+        _ value: sending Value,
+        as: Value.Type = Value.self,
+        eventLoop: EventLoop
+    ) -> NIOLoopBoundBox<Value> {
+        // Here, we -- possibly surprisingly -- do not precondition being on the EventLoop. This is okay for a few
```

Comment:
The comment is a copy-pasta from the other off-EL initializers, but I think it's fair to keep this for consistency.